### PR TITLE
Add option to override regular expression to cli

### LINKED ISSUE #224: regexp cli argument for wordclout_cli.py
I faced some limitations when using wordcloud_cli.py. The default regexp  \w[\w']+ does not match i.e. "c" "c++", "R", "µC", see [1].  I would recommend to register a new optional command line argument and pass it to the WordCloud constructor.

```
parser.add_argument('--regexp', metavar='regexpression', default="\w[\w']+",
help="specify an explicit regular expression, default is \w[\w']+")
wordcloud = wc.WordCloud(..., regexp=regexpression).generate(args.text)
```
[1] http://fiddle.re/60e26n

### PR DESCRIPTION
**Pull request by boidolr**
fixes #224

### CODE REVIEW

File: test/test_wordcloud_cli.py [hunk 1]
```diff
@@ -24,6 +24,7 @@
 ]
 ARGUMENT_SPEC_REMAINING = [
     ArgOption(cli_name='stopwords', init_name='stopwords', pass_value=temp.name, fail_value=None),
+    ArgOption(cli_name='regexp', init_name='regexp', pass_value=r'\w{2,}', fail_value=r'12('),
```
Comment (amueller) [inline test/test_wordcloud_cli.py]: Can you explain the fail_value?

**Comment by amueller** [comment]
could you please add a test for this?

File: test/test_wordcloud_cli.py (hunk 1 above)
Comment (boidolr) [reply test/test_wordcloud_cli.py]: Sure. The `fail_value` is an invalid regular expression as the opening bracket is never closed. If you have another value in mind just replace it.

**Comment by boidolr** [comment]
I assume you'd like a test for the changed word count?

**Comment by amueller** [comment]
any test that suggest that the regex is used appropriately. You could check the dictionary with different regexps

File: test/test_wordcloud_cli.py (hunk 1 above)
Comment (amueller) [reply test/test_wordcloud_cli.py]: oh sorry I misread where this is... never mind, sure, makes sense.

**Comment by amueller** [comment]
thanks :)

**Pull request merged**